### ISSUE TITLE
Farm_Mushroom - unable to add new Mushroom Substrate record

### ISSUE THREAD

**Issue by KonstantinPopov24x777** [farmOS/farmOS#44]
PHP 5.5.5, Apache, FreeBSD, MySQL. FarmOS installed from Drupal pre-packaged distribution, farm_mushroom added from github.
After installation I able to add Substrate Type and Mushroom Species recrords. But when I try to add Mushroom Substrate - I can see only Location and Asset status fields. 
![newsubstrateform](https://user-images.githubusercontent.com/33040759/31929380-798369da-b8a4-11e7-8e5b-69e49468222f.png)
When I try to fill it and Save - error displayed that I have to specify Name, Mushroom Species and Substrate type
![newsubstrateerror](https://user-images.githubusercontent.com/33040759/31929437-b66c8f2a-b8a4-11e7-8f06-d2174891eb4d.png)

**Comment by mstenta** [comment]
Thanks for the report @KonstantinPopov24x777

We talked about it a bit in IRC, but I had to leave.  I'm still not sure what could be causing this... I'd like to get to the bottom of it.

Would you like to try debugging together in IRC again sometime?

**Comment by mstenta** [comment]
Update: @KonstantinPopov24x777 and I debugged this in IRC and discovered that it was a combination of issues. He was using the latest version of the Mushroom module (7.x-1.x), which is only compatible with the development snapshot of farmOS (7.x-1.x). I plan to roll out releases of the contrib modules (including Mushroom) when beta16 of farmOS is released, to prevent this in the future. Also, there was a missing dependency in farmOS which meant that a required module wasn't being enabled (Field Group: Easy Responsive Tabs to Accordion). This was fixed. So once beta16 is released and new versions of Mushroom are tagged this won't affect future users. I plan to do all of that later today. Closing this issue.
